content/km/about-us.md

```diff
@@ -12,6 +12,7 @@ biography: [
 Self-driving Cars"}
 ]
 works: [
+{ "date": "2024-04-01T05:28:04+0000", "to": "បច្ចុប្បន្ន", "note": "Instructor, CADT, Phnom Penh, Cambodia" },
```

Comment:
Thanks for your change.
I'd to make suggestion.
Please official title in Khmer language.

Reply:
How should I put in title in Khmer?

Reply:
@KimangKhenng , Maybe you should use "គ្រូបង្រៀន"

Reply:
Got it, I will change and update my PR